app/src/main/java/uk/ac/ebi/atlas/experimentpage/json/opentargets/EvidenceService.java

```diff
@@ -403,12 +408,18 @@ private JsonObject associationRecord(JsonObject uniqueAssociationFields,
     private ImmutableMap<Contrast, DiseaseAssociation> getDiseaseAssociations(DifferentialExperiment experiment) {
         var contrastToDiseaseBuilder = ImmutableMap.<Contrast, DiseaseAssociation>builder();
         for (var contrast: experiment.getDataColumnDescriptors()) {
-            DiseaseAssociation.tryCreate(experiment, contrast)
+            DiseaseAssociation.tryCreate(getExperimentDesign(experiment.getAccession()), experiment, contrast)
                     .ifPresent(diseaseAssociation -> contrastToDiseaseBuilder.put(contrast, diseaseAssociation));
         }
         return contrastToDiseaseBuilder.build();
     }
 
+    private ExperimentDesign getExperimentDesign(String experimentAccession) {
```

Comment:
I have seen this `getExperimentDesign` method in multiple classes, can we reuse it instead of implementing it again and again.

Reply:
@upendrakumbham Done